src/apis/semanticSearchPostgresDocs.ts

```diff
@@ -8,12 +8,12 @@ const inputSchema = {
   version: z.coerce
     .number()
     .int()
-    .nullable()
-    .describe('The PostgreSQL version to use for the query. Defaults to 17.'),
+    .optional()
```

Comment:
Changing this will break things. See https://github.com/timescale/tiger-slack-mcp-server/pull/13